packages/holocron/src/reactStreaming.js

```diff
@@ -0,0 +1,39 @@
+import { createContext, useContext } from 'react';
+
+const ReactStreamingContext = createContext({});
+
+const ModuleContext = createContext();
```

Comment:
Such a NIT, but `ModuleContext` does not belong to `streaming` as a system, it belongs to `holocronModule` as a system.

(which just means move this one line to holocronModule.jsx and then import it here, rather than the other way arround)

Reply:
Resolved in [30e41f6](https://github.com/americanexpress/holocron/commit/30e41f63daafabf85b9866acf59dfcee55561f9f)